### PR TITLE
added bio

### PR DESCRIPTION
**Pull request by silentsoundz**

### CODE REVIEW

File: js/resumeBuilder.js [hunk 1]
```diff
@@ -5,3 +5,23 @@ var fullName = HTMLheaderName.replace("%data%", fullName);
...
+
+  "welcome":"Greetins and salutations",
+  "skills": skills
+};
+$("#main").append (bio.name);
+$("#main").append (bio.age);
+$("#main").append (bio.role);
+$("#main").append (bio.contacts);
+$("#main").append (bio.welcome);
+$("#main").append (bio.skills);
```
Comment (mantinone) [inline js/resumeBuilder.js]: So it looks like they're having you use jquery to construct a webpage?  
Looks good to me!

Comment (silentsoundz) [inline js/resumeBuilder.js]: thanks!

File: js/resumeBuilder.js [hunk 2]
```diff
@@ -5,3 +5,23 @@ var fullName = HTMLheaderName.replace("%data%", fullName);
 $("#header").append(fullName);
 var role = HTMLheaderRole.replace("%data%", jobTitle);
 $("#header").append (jobTitle);
```
Comment (deadlyicon) [inline js/resumeBuilder.js]: formatting should be `$("#header").append(jobTitle);` no space between method name and parens

File: js/resumeBuilder.js [hunk 3]
```diff
@@ -5,3 +5,23 @@ var fullName = HTMLheaderName.replace("%data%", fullName);
...
+  "contacts": {
+    "mobile": "415-846-4913",
+    "email": "silentsoundz1977@gmail.com",
+    "Location": "Oakland"
+  },
+
+  "welcome":"Greetins and salutations",
+  "skills": skills
+};
+$("#main").append (bio.name);
```
Comment (deadlyicon) [inline js/resumeBuilder.js]: everytime you use the `$` function to find an element its has to look it up in the DOM. Avoid doing this work over and over by storing the result of `$("#main")` in a variable. Or use chaining:

```js
$("#main")
  .append(bio.age)
  .append(bio.role)
  .append(bio.contacts)
  .append(bio.welcome)
  .append(bio.skills);
```


also formatting!

**Pull request merged**